app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoScreen.kt

```diff
@@ -131,4 +135,18 @@ class IntroVideoScreen(
       CrashReporter.report(ActivityNotFoundException("Unable to play simple video because no supporting apps were found."))
     }
   }
+
+  @Parcelize
+  data class Key(
+      val registrationEntry: OngoingRegistrationEntry
+  ) : ScreenKey() {
+
+    @IgnoredOnParcel
+    override val analyticsName = "Onboarding Intro Video"
+
+    @IgnoredOnParcel
+    override val type = ScreenType.FullScreen
```

Comment:
The default type is already `FullScreen` for `ScreenKey`, so we don't need to override this.